python/shotgun_model/data_handler_cache.py

```diff
@@ -130,7 +131,7 @@ def get_children(self, parent_uid):
             cache_node = self._cache[self.CACHE_BY_UID].get(parent_uid)
 
         if cache_node:
-            for item in cache_node[self.CACHE_CHILDREN].itervalues():
+            for item in six.itervalues(cache_node[self.CACHE_CHILDREN]):
```

Comment:
Caches can be really, really big, so we're sticking to itervalues instead of just calling `.values()`